app/modules/ads/setup.js

```diff
@@ -63,6 +63,7 @@ function setupAdContext(adsContext, instantGlobals, isOptedIn = false) {
 
   context.set('options.slotRepeater', isGeoEnabled('wgAdDriverRepeatMobileIncontentCountries'));
   context.set('slots.incontent_boxad_1.adUnit', context.get('megaAdUnitId'));
+  context.set('slots.incontent_player.adUnit', context.get('megaAdUnitId'));
```

Comment:
it looks like adEngine could accept an object with N properties to the set function